javascript/javascript1/week2/eventApp.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document2</title>
+  </head>
+
+  <body>
+    <script>
+
+      /*newDate is JS built-in function to get 
+      today*s date. setDate is JS to get future date.
+      to get future setdate, i add the number of days 
+      to the getDate  */
+
+
+      function getEventWeekday(days) {
+        let targetDate = new Date();
+        targetDate.setDate(targetDate.getDate() + days);
+        console.log(targetDate);
+      }
+      getEventWeekday(5);
```

Comment:
I don't think this quite achieves what the assignment expects. I would take another read through the homework and take some inspiration from your classmates homework to find the correct solution!